lib/govuk_app_config/govuk_error/configuration.rb

```diff
@@ -27,38 +27,38 @@ def before_send=(closure)
   protected
 
     def ignore_exceptions_if_not_in_active_sentry_env
-      ->(error_or_event, _hint) { error_or_event if active_sentry_environments.include?(sentry_environment) }
+      ->(event, _hint) { event if active_sentry_environments.include?(sentry_environment) }
     end
 
     def ignore_excluded_exceptions_in_data_sync
-      lambda { |error_or_event, _hint|
+      lambda { |event, hint|
         data_sync_ignored_error = data_sync_excluded_exceptions.any? do |exception_to_ignore|
           exception_to_ignore = Object.const_get(exception_to_ignore) unless exception_to_ignore.is_a?(Module)
-          exception_chain = Sentry::Utils::ExceptionCauseChain.exception_to_array(error_or_event)
+          exception_chain = Sentry::Utils::ExceptionCauseChain.exception_to_array(hint[:exception])
           exception_chain.any? { |exception| exception.is_a?(exception_to_ignore) }
         rescue NameError
           # the exception type represented by the exception_to_ignore string
           # doesn't even exist in this environment, so won't be found in the chain
           false
         end
 
-        error_or_event unless data_sync.in_progress? && data_sync_ignored_error
+        event unless data_sync.in_progress? && data_sync_ignored_error
       }
     end
 
     def increment_govuk_statsd_counters
-      lambda { |error_or_event, _hint|
+      lambda { |event, hint|
         GovukStatsd.increment("errors_occurred")
-        GovukStatsd.increment("error_types.#{error_or_event.class.name.demodulize.underscore}")
-        error_or_event
+        GovukStatsd.increment("error_types.#{hint[:exception].class.name.demodulize.underscore}")
```

Comment:
Similar comment about nil - we could have error_types.nil_class come through here. I'm not sure what came through in the past if someone used `capture_message` maybe we have some error_types.sentry_event?

By the way, small aside that has probably not been noticed or caused a problem in the past. The `demodularize` method is part of ActiveSupport which isn't a dependency of this Repo. It's a [very simple method](https://github.com/rails/rails/blob/d3f4db9e95a475822c02b29241a5d07cbcff6fd5/activesupport/lib/active_support/inflector/methods.rb#L228-L235) so we can probably just replace it with some inline code rather than requiring activesupport (`"::Test".split("::").last`)